components/eam/cime_config/config_compsets.xml

```diff
@@ -298,39 +317,45 @@
 	<value  compset="C2R4_EAM">2004-01-01</value>
 	<value  compset="AR95_EAM">1995-07-18</value>
 	<value  compset="AR97_EAM">1997-06-19</value>
+	<value  compset="RICO_EAM">2004-12-16</value>
       </values>
     </entry>
 
     <entry id="CALENDAR">
       <values>
 	<value  compset="GEOS_">GREGORIAN</value>
+   <value  compset="RICO_EAM">GREGORIAN</value>
```

Comment:
Make sure the indent matches the other entries here.

Reply:
hmm. my local branch that is up to date with this remote branch gives me the matching indents as shown below. 
![image](https://user-images.githubusercontent.com/40573463/127210364-f80738f8-b19b-42d2-a663-2cf82822013d.png)
I am not sure why it shows different.

Reply:
You might have added a tab character by mistake.

Reply:
yes it was a whitespace issue. I corrected it.